MAiD - Case Management App/force-app/main/default/objects/Account/fields/Other_Profession_Specify__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Other_Profession_Specify__c</fullName>
+    <encryptionScheme>None</encryptionScheme>
```

Comment:
@AmishMistri There is no description for this field. Can you check with Lara in the refinement call today.